share/eds/byzantine/bad_encoding.go

```diff
@@ -143,17 +143,17 @@ func (p *BadEncodingProof) Validate(hdr libhead.Header) error {
 
 	// verify that Merkle proofs correspond to particular shares.
 	shares := make([][]byte, len(merkleRowRoots))
-	for index, share := range p.Shares {
-		if share == nil {
+	for index, shr := range p.Shares {
```

Comment:
Nit: but I think single share is usually `sh`. I don't mind naming it `shr` too, but it would be nice to make it consistent everywhere.

Reply:
Changed everything to shr